scripts/run-integration-tests.sh

```diff
@@ -38,10 +38,10 @@ main() {
     #Check if Fabric is in the gopath. Fabric needs to be in the gopath for the integration tests
     if [ ! -d "${FABRIC_DIR}" ]; then
         echo "Downloading Fabric Branch v1.4.0"
-        git clone https://github.com/hyperledger/fabric $FABRIC_DIR --branch v1.4.0 --single-branch --depth 1
+        git clone https://github.com/hyperledger/fabric $FABRIC_DIR --branch release-1.4 --single-branch --depth 1
     else
         FABRIC_VERSION=$(git -C ${FABRIC_DIR} describe --abbrev=0)
-        if [[ ${FABRIC_VERSION} != "v1.4.0" ]]; then
+        if [[ ${FABRIC_VERSION} != "v1.4.4" ]]; then
```

Comment:
release-1.4 branch is going to float, can we check for the commit with
`git merge-base --is-ancestor 52e51550b0b8f08b2f3c9052e851a82c002d406f HEAD`
hyperledger/fabric@52e51550b0b8f08b2f3c9052e851a82c002d406f